jskit/encoders/utils/evaluate.py

```diff
@@ -1,41 +1,18 @@
-import os
 import torch
 from torch.utils.data import DataLoader
 from . import tokenizer as token_util
-import configparser
-config = configparser.ConfigParser()
 
-max_history, max_contexts_length, max_candidate_length, device = None, \
-    None, None, None
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
```

Comment:
Currently we are setting this in more than one place in the code. Is that necessary? can we consolidate this too>